src/context/constants.ts

```diff
@@ -6,6 +6,8 @@ export const SearchButtonText = 'Искать';
 
 export const SearchInputPlaceholder = 'Поиск';
 
+export const DeleteAll = 'УДАЛИТЬ ВСЕ';
```

Comment:
*just to think about*
https://formatjs.io/docs/getting-started/application-workflow